src/apps/answers/crud/answers.py

```diff
@@ -222,6 +224,56 @@ async def get_flow_submissions(
 
         return parse_obj_as(list[FlowSubmission], data), count
 
+    async def get_applet_submissions(
+        self, applet_id: uuid.UUID, *, page=None, limit=None
+    ) -> tuple[list[AppletSubmission], int]:
+        subjectSchemaRespondent = aliased(SubjectSchema)
+        subjectSchemaTarget = aliased(SubjectSchema)
+
+        query: Query = (
+            select(
+                AnswerSchema.submit_id,
+                AnswerSchema.version,
+                AnswerSchema.applet_id,
+                AnswerSchema.id,
+                AnswerSchema.respondent_id,
+                AnswerSchema.target_subject_id,
+                AnswerSchema.updated_at,
+                AnswerSchema.created_at,
+                subjectSchemaRespondent.tag.label("respondent_tag"),
+                subjectSchemaTarget.tag.label("target_tag"),
+                ActivityHistorySchema.name.label("activity_name"),
+            )
+            .outerjoin(subjectSchemaTarget, AnswerSchema.target_subject_id == subjectSchemaTarget.id)
```

Comment:
I want to remind `answers`, `asnwer_items` tables cannot be joined with other tables because of arbitrary database

Reply:
Good catch! I always forgot about that. Ok I will rewrite everything and get subjects data after query all answers

Reply:
done!